Makefile

```diff
@@ -36,3 +42,6 @@ build-server:
 build-client:
 	cargo build ${CLIENT_FEATURES} --bin ${CLIENT_BIN}
 
+.PHONY: gen-cert
+gen-cert:
```

Comment:
Super nit (for bonus points!): Have an actual make target for the cert (not a `PHONY` one) so it generates it automatically if it doesn't exist when `make serve` is run

Reply:
That is a GREAT idea.

Reply:
Done.